ibis/backends/tests/tpc/queries/duckdb/ds/75.sql

```diff
@@ -13,7 +13,7 @@ WITH all_sales AS
              i_category_id ,
              i_manufact_id ,
              cs_quantity - COALESCE(cr_return_quantity,0) AS sales_cnt ,
-             cs_ext_sales_price - COALESCE(cr_return_amount,0.0) AS sales_amt
+             cs_ext_sales_price - COALESCE(cr_return_amount,CAST(0.0 AS DECIMAL(7,2))) AS sales_amt
```

Comment:
This seems like it might be a red herring, since the literal value of the operation passes:

```
SELECT coalesce(CAST(0., 'Decimal(17, 2)'), 0., CAST(0., 'Decimal(7, 2)'))

Query id: 907baa21-6411-4134-a53f-a068ac26ca46

   ┌─coalesce(CAST(0., 'Decimal(17, 2)'), 0., CAST(0., 'Decimal(7, 2)'))─┐
1. │                                                                   0 │
   └─────────────────────────────────────────────────────────────────────┘
```

Reply:
We're probably inserting casts in Ibis where we shouldn't be.

Reply:
Lemme poke around and see if I can remove those.

Reply:
Ok, it seems like ClickHouse is handling the literal case

😮‍💨

Reply:
This is fine as is, since it's ClickHouse's problem

Reply:
yeah, not very satisfying, I know